### PR TITLE
S 2

### PR DESCRIPTION
**Pull request by sapkotashardul**

### CODE REVIEW

File: src/app/google_api.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    '''
+    For the app to make Google Calendar based requests.
+    '''
+    def __init__(self, activity: Enum, start=datetime(year=2018, month=11, day=1, hour=22), end=datetime(year=2018, month=11, day=1, hour=23)):
+        self.activity = activity
+        self.start = start
+        self.end = end
+        self.scope = 'https://www.googleapis.com/auth/calendar'
+
+    def get_credentials(self):
```
Comment (varuntaak) [inline src/app/google_api.py]: This should be a dependency as our application won't read anything from the file system. We will store the login session (google api credentials) in a session variable. So please take this out and inject it. Ideally, for now, you can move this function to a test case.

File: src/app/google_api.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                ],
+            },
+        }
+
+        event = service.events().insert(calendarId='primary', body=event).execute()
+        'Event created: %s' % (event.get('htmlLink'))
+
+
+if __name__ == '__main__':
+    g = GoogleAPI(activity=Social.COFFEE.name, start=datetime(year=2018, month=11, day=3, hour=21).isoformat(), end=datetime(year=2018, month=11, day=3, hour=22).isoformat())
```
Comment (varuntaak) [inline src/app/google_api.py]: move this to a test case. This would be an integration test looks like a unit test, as it is extending to google api.

**Comment by sapkotashardul** [comment]
I've now made the changes based on your suggestion

**Comment by varuntaak** [comment]
@sapkotashardul  I merged it to for now to integrate end-to-end, I will refactor it as and when necessary.

**Pull request merged**